bands/templates/band_admin/band_info.html

```diff
@@ -11,183 +11,184 @@
 
 <h2>Band Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
 
+<br/> <br/>
 <form class="form-horizontal" role="form" method="post" action="band_info">
     {% csrf_token %}
     <input type="hidden" name="id" value="{{ band.id }}" />
 
-    {% if band.panel_name %}
-        <div class="row">
-            <label class="col-sm-2 control-label">Panel Name:</label>
-            <div class="col-sm-6">
-                {{ band.panel_name }}
-            </div>
+    {% if band.panel.name %}
```

Comment:
Silly question that I really ought to know the answer to off the top of my head after using Django templates for ~10 years: does this raise an exception if ``band.panel`` is ``None``?  I know that saying ``{{ foo.whatever.nullable.bar }}`` is fine, because by default Django will just swallow exception and replace them with the empty string.  But I don't think it does that for ``{% if %}`` statements, though I might be wrong.